src/main/kotlin/dev/fakek/fakes/FakeCommerce.kt

```diff
@@ -0,0 +1,12 @@
+package dev.fakek.fakes
+
+/**
+ * FakeCommerce provides a commerce with random information.
+ *
+ * @param address a instance of FakeAddress containing all info related to the fake address of the commerce.
+ */
+class FakeCommerce(
```

Comment:
Could this be updated to a `data` class and then pass in `FakeCompany` as an additional property?

Reply:
sure